lib/charms/mimir_coordinator_k8s/v0/mimir_cluster.py

```diff
@@ -91,13 +92,30 @@ def dump(self, databag: Optional[MutableMapping[str, str]] = None, clear: bool =
         if nest_under:
             databag[nest_under] = self.json()
 
-        dct = self.model_dump()
+        dct = self.model_dump(by_alias=True)
         for key, field in self.model_fields.items():  # type: ignore
             value = dct[key]
             databag[field.alias or key] = json.dumps(value)
-
         return databag
 
+    def update(self, **data: Any):
```

Comment:
do we still need this?

Reply:
Good point, I don't think so; removing it!